src/Raven.Client/Documents/Operations/OperationState.cs

```diff
@@ -46,4 +46,13 @@ public enum OperationStatus
         Faulted,
         Canceled
     }
+    
+    public interface IOperationProcessedDetails
```

Comment:
Do we need this interface?

Reply:
Can do without it.
I used in 2 places: ` class BulkInsertProgress` & `class BulkOperationResult`
@ppekrol  should I remove it?

Reply:
Yeah, let's remove it.

Reply:
done